### DISCUSSION TITLE
Should have id in class UserProfile in module @loopback/sercurity

### DISCUSSION THREAD

**Discussion by Vn-ChemGio** [loopbackio/loopback-next#8034] [Q&A]
In most of case we want to get id of user to use in other module or class , when we want to get current User login
Example: In my case , I'm trying to extend module loopback-soft-delete
Currently I'm use    
    "@loopback/security": "^0.5.4",
    "@loopback/authentication": "^7.3.4",
    "@loopback/authentication-jwt": "^0.9.4",
to auth user, and update some meta data but I can get id of user from UserProfile
So, please add "id" field to UserProfile class

**Comment by Dherlou** [comment]
s. https://github.com/loopbackio/loopback-next/issues/3846#issuecomment-566721530
